cypress/integration/multi_select_dialog.js

```diff
@@ -42,17 +57,42 @@ context('MultiSelectDialog', () => {
 
 		cy.get_open_dialog()
 			.get(`.frappe-control[data-fieldname="allow_child_item_selection"]`)
+			.find('input[data-fieldname="allow_child_item_selection"]')
 			.should('exist')
-			.click();
+			.click({force: true});
 
 		cy.get_open_dialog()
 			.get(`.frappe-control[data-fieldname="child_selection_area"]`)
 			.should('exist');
 
 		cy.get_open_dialog()
-			.get(`.dt-row-header`).should('contain', 'Assignment Rule');
+			.get(`.dt-row-header`).should('contain', 'Contact');
 
 		cy.get_open_dialog()
-			.get(`.dt-row-header`).should('contain', 'Day');
+			.get(`.dt-row-header`).should('contain', 'Email Id');
+
+		cy.get_open_dialog()
+			.get(`.dt-row-header`).should('contain', 'Is Primary');
+	});
+
+	it('tests more button', () => {
+		cy.get_open_dialog()
+			.get(`.frappe-control[data-fieldname="more_btn"]`)
+			.should('exist')
+			.as('more-btn');
+		
+		cy.get_open_dialog().get('.list-item-container').should(($rows) => {
+			expect($rows).to.have.length(20);
+		});
+
+		cy.get('@more-btn').find('button').click({force: true});
+		cy.wait(1000);
```

Comment:
Wait till the network request is fulfilled instead of waiting for an arbitrary duration to avoid a flaky test.

Something like https://github.com/frappe/frappe/blob/develop/cypress/integration/report_view.js#L27